src/Client/WorkflowClient.php

```diff
@@ -276,17 +277,24 @@ public function listWorkflowExecutions(
                 $request->setNextPageToken($nextPageToken);
             } while ($nextPageToken !== '');
         };
-        $counter = function () use ($namespace, $query): int {
-            $response = $this->client->CountWorkflowExecutions((new CountWorkflowExecutionsRequest())
-                ->setNamespace($namespace)
-                ->setQuery($query));
-
-            return (int)$response->getCount();
-        };
+        $counter = fn(): int => (int)$this->countWorkflowExecutions($query, $namespace)->getCount();
 
         return Paginator::createFromGenerator($loader($request), $counter);
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function countWorkflowExecutions(
+        string $query,
+        string $namespace = 'default',
+    ): CountWorkflowExecutionsResponse {
```

Comment:
IMO you should return a high-level wrapper around the response instead of the raw gRPC response. In Go we regret using raw proto responses and in other SDKs we do not.

Reply:
Ugh, i specifically decided to go with GRPC one since it contains a single value. @roxblnfk sorry about that.